docs/installation.rst

```diff
@@ -260,6 +260,15 @@ Make sure those are installed before attempting the ScanCode.io installation::
 See also `ScanCode-toolkit Prerequisites <https://scancode-toolkit.readthedocs.io/en/
 latest/getting-started/install.html#prerequisites>`_ for more details.
 
+For the ``CollectSymbols`` pipeline, `Universal Ctags <https://github.com/universal-ctags/ctags>`_ is needed.
```

Comment:
Use a proper reference for ``CollectSymbols`` once added in the  built-in-pipelines.rst file.

Reply:
Done